Dockerfile

```diff
@@ -21,6 +21,7 @@ ENV DEBIAN_FRONTEND noninteractive
 CMD ["/bin/bash"]
 
 ARG WGET_VERSION="1.21.3-1+b2"
+ARG OPENJDK_17_VERSION="17.0.11+9-1~deb12u1"
```

Comment:
My concern is:
* This DockerFile is probably broken building locally on ARM without knowing argument magic
* We don't have comments / it's not clear that we might need to update both the ARM version when this is updated

Perhaps we should instead use `TARGETPLATFORM` and then encode the two versions inside the docker file, for better locality, and to ensure they work without magic build parameters? e.g. https://nielscautaerts.xyz/making-dockerfiles-architecture-independent.html

For this release, maybe we can just pull in `17.0`, and build something better on a separate PR?

Reply:
let's do it in https://radixdlt.atlassian.net/browse/NODE-635